src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -92,7 +92,7 @@ public static function createChoiceName(object $choice, $key, string $value): st
      * @internal This method is public to be usable as callback. It should not
```

Comment:
Unrelated, but this one, the one above and the one in EntityType below can also be made `private` in Symfony 5.4 I think. The required `public` visibility is from 6 years ago, when `$this` inside a closure was not yet supported (https://github.com/symfony/symfony/commit/3172d73b6c7969fe2f01664c495c641256418f39).

Reply:
correct, any up for a PR doing so?